src/Apps/Auction/Components/RegistrationForm.tsx

```diff
@@ -0,0 +1,293 @@
+import { Box, Button, Flex, Input, Serif } from "@artsy/palette"
+import { AuctionApp_sale } from "__generated__/AuctionApp_sale.graphql"
+import { RegistrationFormCreateBidderMutation } from "__generated__/RegistrationFormCreateBidderMutation.graphql"
+import { RegistrationFormCreateCreditCardMutation } from "__generated__/RegistrationFormCreateCreditCardMutation.graphql"
+import { ConditionsOfSaleCheckbox } from "Apps/Auction/Components/ConditionsOfSaleCheckbox"
+import { CreditCardInput } from "Apps/Order/Components/CreditCardInput"
+import { Form, Formik, FormikProps } from "formik"
+import React from "react"
+import { commitMutation, graphql, RelayProp } from "react-relay"
+import {
+  CardElement,
+  injectStripe,
+  ReactStripeElements,
+} from "react-stripe-elements"
+import { data as sd } from "sharify"
+import styled from "styled-components"
+import * as Yup from "yup"
+
+export const StyledCardElement = styled(CardElement)`
+  width: 100%;
+  padding: 9px 10px;
+`
+
+interface FormValues {
+  name: string
+  street1: string
+  city: string
+  state: string
+  credit_card: string
+  postal_code: string
+  telephone: string
+  agree_to_terms: boolean
+}
+
+const InnerForm = (props: FormikProps<FormValues>) => {
+  const {
+    touched,
+    errors,
+    handleBlur,
+    isSubmitting,
+    handleChange,
+    values,
+  } = props
+
+  return (
+    <Form>
+      <Box mt={4}>
+        <Serif weight="semibold" size="4t" mb={2}>
+          Card Information
+        </Serif>
+        <Box mb={2}>
+          <Input
+            onBlur={handleBlur}
+            onChange={handleChange}
+            error={touched.name && errors.name}
+            required
+            title="Name on card"
+            placeholder="Add full name"
+            name="name"
+          />
+        </Box>
+
+        <Box mb={2}>
+          <Serif size="3t" mb={0.5}>
+            Credit card number*
+          </Serif>
+          <CreditCardInput
+            error={{ message: errors.credit_card } as stripe.Error}
+          />
+        </Box>
+      </Box>
+      <Box mt={4}>
+        <Serif weight="semibold" size="4t">
```

Comment:
@dblandin was there a reason to not re-use `AddressForm` here? (here's an [example](https://github.com/artsy/reaction/blob/master/src/Components/Payment/PaymentForm.tsx#L124) of where we're using both `AddressForm` and `CreditCardInput` in a custom wrapper).